sdl/Makefile.linux

```diff
@@ -45,11 +45,12 @@ clean:
 	rm $(OBJECTS) $(OBJECTS:.o=.d) $(TARGET)
 
 install:
-	mkdir -p $(PREFIX)/share/man/man1/
-	../docs/hivelytracker.1 $(PREFIX)/share/man/man1/
+	install -Dm644 ../Docs/hivelytracker.1 $(PREFIX)/share/man/man1/hivelytracker.1
 	gzip -9 $(PREFIX)/share/man/man1/hivelytracker.1
-	install -o root -g root -m 755 ../sdl/hivelytracker $(PREFIX)/bin
-	install -o root -g root -m 644 ../sdl/winicon.png $(PREFIX)/share/icons/hivelytracker.png
-	install -o root -g root -m 644 ../sdl/hivelytracker.desktop $(PREFIX)/share/applications
-	install -o root -g root -m 644 ../Instruments $(PREFIX)share/hivelytracker                                                                            
-	install -o root -g root -m 644 ../Skins $(PREFIX)/share/hivelytracker
+	install -Dm 755 ../sdl/hivelytracker $(PREFIX)/bin/hivelytracker
+	install -Dm 644 ../sdl/winicon.png $(PREFIX)/share/icons/hivelytracker.png
+	install -Dm 644 ../sdl/hivelytracker.desktop $(PREFIX)/share/applications/hivelytracker.desktop

```

Comment:
One could install hivelytracker as a normal user, so I'm dropping "root". I'm also using -D since the paths may not exist